optaplanner-examples/src/test/java/org/optaplanner/examples/common/app/SolverPerformanceTest.java

```diff
@@ -43,13 +50,35 @@
  * Always use a {@link Test#timeout()} on {@link Test}, preferably 10 minutes because some of the Jenkins machines are old.
  * @param <Solution_> the solution type, the class with the {@link PlanningSolution} annotation
  */
+@RunWith(Parameterized.class)
 public abstract class SolverPerformanceTest<Solution_> extends LoggingTest {
 
-    private static final String MOVE_THREAD_COUNT_OVERRIDE = System.getProperty(TestSystemProperties.MOVE_THREAD_COUNT);
+    private static final String MOVE_THREAD_COUNTS_STRING = System.getProperty(TestSystemProperties.MOVE_THREAD_COUNTS);
+
+    private final String moveThreadCount;
 
     protected SolutionFileIO<Solution_> solutionFileIO;
     protected String solverConfig;
 
+    @Parameterized.Parameters(name = "{index}: {0}")
+    public static Collection<Object[]> getSolutionFilesAsParameters() {
+        List<Object[]> testParams = new ArrayList<>();
```

Comment:
When fixing the merge conflict on the new flight crew example I wanted to avoid the call to new ArrayList() without a capacity. One thing led to another and I ended up using streams:
  https://github.com/kiegroup/optaplanner/pull/412/commits/f57313b47e83d915b181cfa69681e819963a2cb3